package.json

```diff
@@ -157,7 +157,7 @@
     "pm2-runtime": "./bin/pm2-runtime"
   },
   "dependencies": {
-    "async": "1.5",
+    "async": "^2.4.1",
```

Comment:
okay, I will debug why the tests are failing, any pointers? it seems like the tests are only failing on `node@7`